azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/database/DatabaseService.java

```diff
@@ -96,6 +109,27 @@ public void restoreNamespaceRoleAccess(String namespaceRole) {
     databaseDao.restoreLoginPrivileges(namespaceRole);
   }
 
+  private SecretKey decodeBase64Key(String encryptionKeyBase64) {
+    byte[] decodedKey = Base64.getDecoder().decode(encryptionKeyBase64);
+    return new SecretKeySpec(decodedKey, 0, decodedKey.length, "AES");
+  }
+
+  private OutputStream encryptIntoOutputStream(OutputStream origin, String encryptionKeyBase64)
+      throws NoSuchPaddingException, NoSuchAlgorithmException, InvalidKeyException {
+    SecretKey encryptionKey = decodeBase64Key(encryptionKeyBase64);
+    Cipher c = Cipher.getInstance("AES");
```

Comment:
You are using AES without settings, you should define the mode and the padding to enhance the security. Recommended `Cipher c = Cipher.getInstance("AES/CBC/PKCS5Padding")`